alpaca_data_cleaned.json

```diff
@@ -222082,7 +222082,7 @@
     {
         "instruction": "Compute the mean and standard deviation for this set of numbers.",
         "input": "2, 3, 7, 5",
-        "output": "Mean = 4.5, Standard deviation = 2.121320344"
+        "output": "Mean = 4.25, Standard deviation = 1.8306"
```

Comment:
Sample standard deviation should be 2.217355783 (as calculated by my scientific calculator). Fun fact, chatgpt3 got this question wrong too

Reply:
Sorry it should be population standard deviation as the output given that the instruction does not specify where those numbers come from. This answer is still wrong though, population SD = 1.920286437